web-local/.jest/setup.js

```diff
@@ -0,0 +1,17 @@
+import { spawn } from "child_process";
```

Comment:
We should move this to package.json and avoid dependancy on jest's features. Something like `setup_binary.sh && (run tests || cleanup_binary.sh)`

Reply:
Not sure I follow - why do you want to avoid using jest's features? Isn't that the point of having a testing framework?

We are already using jest's features, see our existing jest config https://github.com/rilldata/rill-developer/blob/main/web-local/jest.config.json#LL17C2-L20C5

Reply:
I think we should be moving to vitest. It is a lot better since we are already using vite.